src/android/java/io/jxcore/node/JXcoreExtension.java

```
@@ -4,37 +4,30 @@
 public class JXcoreExtension {
 
-    public static String EVENTSTRING_PEERAVAILABILITY   = "peerAvailabilityChanged";
-    public static String EVENTVALUESTRING_PEERID        = "peerIdentifier";
-    public static String EVENTVALUESTRING_PEERNAME      = "peerName";
-    public static String EVENTVALUESTRING_PEERAVAILABLE = "peerAvailable";
+    public final static String EVENTSTRING_PEERAVAILABILITY   = "peerAvailabilityChanged";
+    public final static String EVENTVALUESTRING_PEERID        = "peerIdentifier";
+    public final static String EVENTVALUESTRING_PEERNAME      = "peerName";
+    public final static String EVENTVALUESTRING_PEERAVAILABLE = "peerAvailable";
 
-    public static String EVENTSTRING_NETWORKCHANGED     = "networkChanged";
-    public static String EVENTVALUESTRING_REACHABLE     = "isReachable";
-    public static String EVENTVALUESTRING_WIFI          = "isWiFi";
+    public final static String EVENTSTRING_NETWORKCHANGED     = "networkChanged";
+    public final static String EVENTVALUESTRING_REACHABLE     = "isReachable";
+    public final static String EVENTVALUESTRING_WIFI          = "isWiFi";
 
-    public static String METHODSTRING_SHOWTOAST         = "ShowToast";
+    public final static String METHODSTRING_SHOWTOAST         = "ShowToast";
 
-    public static String METHODSTRING_STARTBROADCAST    = "StartBroadcasting";
-    public static String METHODSTRING_STOPBROADCAST     = "StopBroadcasting";
-
-    public static String METHODSTRING_CONNECTTOPEER     = "Connect";
-    public static String METHODSTRING_DISCONNECTPEER    = "Disconnect";
-    public static String METHODSTRING_KILLCONNECTION    = "KillConnection";
+    public final static String METHODSTRING_STARTBROADCAST    = "StartBroadcasting";
+    public final static String METHODSTRING_STOPBROADCAST     = "StopBroadcasting";
 
+    public final static String METHODSTRING_CONNECTTOPEER     = "Connect";
+    public final static String METHODSTRING_DISCONNECTPEER    = "Disconnect";
```

In METHODSTRING_STARTBROADCAST if the wrong number of arguments are passed in then the function will silently fail and never return an error. Please fix.